docs/src/app_dev/app_val_tokenomics/cc_xfer_nomics.rst

```diff
@@ -0,0 +1,82 @@
+..
+   Copyright (c) 2024 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+..
+   SPDX-License-Identifier: Apache-2.0
+
+.. _cc_xfer_nomics:
+
+Canton Coin Transfer
+====================
+
+A CC ``Amulet`` `Transfer <https://docs.dev.sync.global/app_dev/api/splice-amulet/Splice-AmuletRules.html#type-splice-amuletrules-transfer-72721>`__
+or `TwoStepTransfer <https://docs.dev.sync.global/app_dev/api/splice-amulet/Splice-Amulet-TwoStepTransfer.html#splice-amulet-twosteptransfer>`__
+is submitted as part of an application's business logic and it includes
+a ``sender`` and a ``receiver`` or ``provider`` party. Typically, the application that
+facilitated that transfer will put its own application party ID as the
+``sender``.
+
+Consider a single, simple CC ``TwoStepTransfer`` with: a single ``sender``
+party, a single ``receiver`` party, one application provider, and no
+beneficiaries:
+
+      1. A `ValidatorRewardCoupon <https://docs.dev.sync.global/app_dev/api/splice-amulet/Splice-Amulet.html#type-splice-amulet-validatorrewardcoupon-76808>`__
+         is created. The operator party of the validator that hosts the
+         sender can mint the
+         ``ValidatorRewardCoupon``
+         in the minting step. This hosting relationship is tracked in the
+         `ValidatorRight <https://docs.dev.sync.global/app_dev/api/splice-amulet/Splice-Amulet.html#type-splice-amulet-validatorright-15964>`__\ ``(sender,
+         operator)`` contract.
+
+      2. An `AppRewardCoupon <https://docs.dev.sync.global/app_dev/api/splice-amulet/Splice-Amulet.html#type-splice-amulet-apprewardcoupon-57229>`__
+         is created. By default, the ``provider`` can mint CC in the minting
+         step. However, if a ``beneficiary`` is specified then they will
+         receive the minted CC instead of the ``provider``.
+         The
+         `AppRewardCoupon <https://docs.dev.sync.global/app_dev/api/splice-amulet/Splice-Amulet.html#type-splice-amulet-apprewardcoupon-57229>`__
+         is created with the DSO as the ``signatory`` and the ``provider`` field
+         as an observer. The application provider can set the ``provider`` to
```

Comment:
" The application provider can set the ``provider`` to its application provider ID" 

This is not true in this generality, as the `provider` is an authorizer of the `AmuletRules_Transfer` choice. To be listed as the provider you must use Daml models that funnel your authority to the call of the `AmuletRules_Transfer` choice.

Reply:
Done